.github/workflows/unity.yml

```diff
@@ -0,0 +1,249 @@
+    - name: build_win64_dracodec_unity
+      run: cmake --build build_win_64 --config MinSizeRel --target dracodec_unity
+    - name: build_win64_dracoenc_unity
+      run: cmake --build build_win_64 --config MinSizeRel --target dracoenc_unity
+
+    - name: configure_win32
+      run: cmake . -G "Visual Studio 16 2019" -A Win32 -Bbuild_win_32 -DDRACO_UNITY_PLUGIN=ON -DDRACO_GLTF=ON -DDRACO_BACKWARDS_COMPATIBILITY=OFF
+    - name: build_win32
+      run: cmake --build build_win_32 --config MinSizeRel --target dracodec_unity
+
+    # Universal Windows Platform
+    - name: configure_uwp_arm
+      run: cmake . -G "Visual Studio 16 2019" -A ARM -Bbuild_uwp_arm -DDRACO_UNITY_PLUGIN=ON -DDRACO_GLTF=ON -DDRACO_BACKWARDS_COMPATIBILITY=OFF -DCMAKE_SYSTEM_NAME:String=WindowsStore -DCMAKE_SYSTEM_VERSION:String="10.0"
+    - name: build_uwp_arm
+      run: cmake --build build_uwp_arm --config MinSizeRel --target dracodec_unity
+
+    - name: configure_uwp_arm64
+      run: cmake . -G "Visual Studio 16 2019" -A ARM64 -Bbuild_uwp_arm64 -DDRACO_UNITY_PLUGIN=ON -DDRACO_GLTF=ON -DDRACO_BACKWARDS_COMPATIBILITY=OFF -DCMAKE_SYSTEM_NAME:String=WindowsStore -DCMAKE_SYSTEM_VERSION:String="10.0"
+    - name: build_uwp_arm64
+      run: cmake --build build_uwp_arm64 --config MinSizeRel --target dracodec_unity
+
+    - name: configure_uwp_x86
+      run: cmake . -G "Visual Studio 16 2019" -A Win32 -Bbuild_uwp_x86 -DDRACO_UNITY_PLUGIN=ON -DDRACO_GLTF=ON -DDRACO_BACKWARDS_COMPATIBILITY=OFF -DCMAKE_SYSTEM_NAME:String=WindowsStore -DCMAKE_SYSTEM_VERSION:String="10.0"
+    - name: build_uwp_x86
+      run: cmake --build build_uwp_x86 --config MinSizeRel --target dracodec_unity
+
+    - name: configure_uwp_x64
+      run: cmake . -G "Visual Studio 16 2019" -A x64 -Bbuild_uwp_x64 -DDRACO_UNITY_PLUGIN=ON -DDRACO_GLTF=ON -DDRACO_BACKWARDS_COMPATIBILITY=OFF -DCMAKE_SYSTEM_NAME:String=WindowsStore -DCMAKE_SYSTEM_VERSION:String="10.0"
+    - name: build_uwp_x64
+      run: cmake --build build_uwp_x64 --config MinSizeRel --target dracodec_unity
+
+    - name: package Windows
+      run: |
+        mkdir draco_win/x86
+        mkdir draco_win/x86_64
+        mkdir draco_win/WSA/ARM
+        mkdir draco_win/WSA/ARM64
+        mkdir draco_win/WSA/x86
+        mkdir draco_win/WSA/x64
+        mv build_win_32\MinSizeRel\dracodec_unity.dll draco_win/x86
+        mv build_win_64\MinSizeRel\dracodec_unity.dll draco_win/x86_64
+        mv build_win_64\MinSizeRel\dracoenc_unity.dll draco_win/x86_64
+        mv build_uwp_arm\MinSizeRel\dracodec_unity.dll draco_win/WSA/ARM
+        mv build_uwp_arm64\MinSizeRel\dracodec_unity.dll draco_win/WSA/ARM64
+        mv build_uwp_x86\MinSizeRel\dracodec_unity.dll draco_win/WSA/x86
+        mv build_uwp_x64\MinSizeRel\dracodec_unity.dll draco_win/WSA/x64
+    - name: upload artifact
+      uses: actions/upload-artifact@v2
+      with:
+        name: draco_win
+        path: draco_win
+
+
+  linux:
+
+    runs-on: ubuntu-18.04
```

Comment:
Why not use ubuntu-latest?

Reply:
I've had a users report an [issue](https://github.com/atteneder/DracoUnity/issues/8) when using a library built on ubuntu-latest (20.10 at the time) on ubuntu 18.04 (correct glibc version was not found).

Is there a better way to achieve backwards compatibility? Or cross distribution compatibility in general?